analyzer/src/main/kotlin/managers/SpdxDocumentFile.kt

```diff
@@ -85,25 +85,37 @@ class SpdxDocumentFile(
 
     /**
      * Return the dependencies of [pkg] defined in [doc] of the [SpdxRelationship.Type.DEPENDENCY_OF] type.
+     * Identified dependencies are mapped to ORT [Package]s and are added to [packages], using [workingDir] to define
+     * the version control system directory of the dependencies.
```

Comment:
There is nothing "version control system" specific about `workingDir`, it could be any local directory, also not under version control. So this and similar parts of the sentence below should be removed.